.github/workflows/main.yml

```diff
@@ -206,20 +211,13 @@ jobs:
             -DXPYT_EMSCRIPTEN_WASM_BUILD=ON \
             ..
 
-          make -j5
-
-          empack pack env --env-prefix $PREFIX --outname python_data --config $EMPACK_PREFIX/share/empack/empack_config.yaml
+          make -j5 
 
-          popd
-
-          # Patch output
-          python wasm_patches/patch_it.py
```

Comment:
Is the patch not needed anymore?

Reply:
To be honest, I am not 100% sure. At least for the pure pyjs version I use in the pyjs-code-runner does require this patch. 
We can also leave it in.
What is the intent of uploading the files? are the currently  consumed by any of our tools?

Reply:
Ok,then we leave it in, but I make an issue to check if we actually still need it

Reply:
> That was to be able to test them manually in jupyterlite-xeus-python prior to releasing, which was actually not possible without the relocate feature. But it may be possible now?

So the `libpython` we get in the ci has the prefix `$MAMBA_ROOT_PREFIX/envs/xeus-python-wasm-host` ie smth like `/home/runner/micromamba/envs/xeus-python-wasm-host`. So the generated xeus wasm will have this prefix baked in.

So if you want to use these artifacts you need to create the env with `micromamba` and relocated it to
`/home/runner/micromamba/envs/xeus-python-wasm-host`
This might work.

Reply:
https://github.com/jupyter-xeus/xeus-python/issues/589

@martinRenou  btw I supsect if you just pass the `relocate-prefix` argument  `empack pack env` it will just work. even if the env was not relocated!  This can work because I explicitly set `PYTHONHOME` to the relocated prefix in pyjs
https://github.com/emscripten-forge/pyjs/blob/main/include/pyjs/pre_js/init.js#L25-L26